src/additional-functions/create-modal-input-controls.js

```diff
@@ -209,24 +209,22 @@ export const modalViewData = (
             break;
           case "radio":
             const curInputOptions = inputs[y][4]
-            if (curInputOptions) {
+            if (selected) {
```

Comment:
Can you pls confirm these updates do not break already implemented radio controls?

Reply:
I can confirm these changes do not break already implemented radio controls. I added a 5th index to the input controls.
The Online Zero/Online Upscale/Offline Zero/Offline Upscale APS Indicators for Online Offline Calibration use this 5th index for a default value of yes for the radio button selection.

Reply:
Ok, thanks,